Remotion/Web/Core/UI/SmartPageImplementation/SmartPageInfo.cs

```diff
@@ -54,6 +54,7 @@ public enum ResourceIdentifier
     }
 
     public static readonly string CacheDetectionID = "SmartPage_CacheDetectionField";
+    private const string c_smartPageTokenID = "SmartPage_InstanceToken";
```

Comment:
```suggestion
    private const string c_smartPageTokenID = "SmartPage_TokenField";
```

`...Field` for consistency reason. 
`Instance` or not, that's just a preference thing and since we don't use `Instance` in the names, it feels cleaner this way (no need to think about why the variable name and the value differ) but either one is fine.